### ISSUE TITLE
Configuration broken when upgrading from 2.0.0 to 2.0.1

### ISSUE THREAD

**Issue by chalasr** [FriendsOfPHP/PHP-CS-Fixer#2528]
Hello,
Here is my config:

```php
// .php_cs
$finder = PhpCsFixer\Finder::create()
    ->in([__DIR__.'/src', __DIR__.'/tests', __DIR__.'/bin'])
;

return PhpCsFixer\Config::create()
    ->setUsingCache(false)
    ->setFinder($finder)
    ->setRules([
        '@Symfony' => true,
        'psr0' => false,
        'unalign_equals' => false,
        'unalign_double_arrow' => false,
        'ordered_imports' => true,
        'array_syntax' => ['syntax' => 'short'],
        'header_comment' => [
            'header' => '[header]',
            'commentType' => PhpCsFixer\Fixer\Comment\HeaderCommentFixer::HEADER_COMMENT,
        ],
    ])
;
```

It runs fine on 2.0.0, but since last composer update providing 2.0.1:

>   [PhpCsFixer\ConfigurationException\InvalidConfigurationException (16)]
  The rules contain unknown fixers (unalign_equals, unalign_double_arrow).

Is it a merge mistake? Fabbot.io seems to be broken too, see last PRs on symfony/symfony.
Thanks

**Comment by SpacePossum** [comment]
That is odd as the fixers have been removed in 2.0
(these are no part of `binary_operator_spaces` through configuration of that fixer);
Details can be found here:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.0.0/UPGRADE.md#renamed-rules

Looking at your configuration you can remove both the lines;
```
  'unalign_equals' => false,
  'unalign_double_arrow' => false,
```

as these are the default configuration values for the `binary_operator_spaces` which is part of the `@Symfony` set (which is all so part of your config).

**Comment by keradus** [comment]
```
keradus@keradus:~/github/FriendsOfPHP/PHP-CS-Fixer$ git co v2.0.0-alpha
HEAD is now at d0d76b43... minor #1875 renamed Composer package (fabpot)
keradus@keradus-workLap:~/github/FriendsOfPHP/PHP-CS-Fixer$ php php-cs-fixer fix --rules=unalign_equals
Loaded config from "/home/keradus/github/FriendsOfPHP/PHP-CS-Fixer/.php_cs.dist".
^C
keradus@keradus:~/github/FriendsOfPHP/PHP-CS-Fixer$ git co v2.0.0-beta
Previous HEAD position was d0d76b43... minor #1875 renamed Composer package (fabpot)
HEAD is now at 962b2c53... prepared the 2.0.0-beta release
keradus@keradus-workLap:~/github/FriendsOfPHP/PHP-CS-Fixer$ php php-cs-fixer fix --rules=unalign_equals
Loaded config default from "/home/keradus/github/FriendsOfPHP/PHP-CS-Fixer/.php_cs.dist".

                                         
  [UnexpectedValueException]             
  Rule "unalign_equals" does not exist.  
                                         

fix [--path-mode PATH-MODE] [--allow-risky ALLOW-RISKY] [--config CONFIG] [--dry-run] [--rules RULES] [--using-cache USING-CACHE] [--cache-file CACHE-FILE] [--diff] [--format FORMAT] [--] [<path>]...

keradus@keradus:~/github/FriendsOfPHP/PHP-CS-Fixer$ git co v2.0.0
Previous HEAD position was 962b2c53... prepared the 2.0.0-beta release
HEAD is now at f3baf72e... prepared the 2.0.0 release
keradus@keradus-workLap:~/github/FriendsOfPHP/PHP-CS-Fixer$ php php-cs-fixer fix --rules=unalign_equals
Loaded config default from "/home/keradus/github/FriendsOfPHP/PHP-CS-Fixer/.php_cs.dist".

                                         
  [UnexpectedValueException]             
  Rule "unalign_equals" does not exist.  
                                         

fix [--path-mode PATH-MODE] [--allow-risky ALLOW-RISKY] [--config CONFIG] [--dry-run] [--rules RULES] [--using-cache USING-CACHE] [--cache-file CACHE-FILE] [--diff] [--format FORMAT] [--] [<path>]...

```


rule was available on v2.0.0-alpha, but not beta or stable release.

for fabbot, ask it's maintainer
